securedrop/source_templates/base.html

```diff
@@ -30,10 +30,6 @@
         {% endblock %}
 
         <div class="panel selected">
-        {% if 'logged_in' in session %}
```

Comment:
The logout button was moved from the base to lookup template as part of the layout changes. One consequence of this is that it no longer appears on other pages that use extend `base.html` - `/why-journalist-key` and the error pages being the most likely examples. It might be better to have the logout widget as an include, and include it on pages that may be rendered during active session.

Reply:
I actually didn't need to move the logout button, only ensure the HR that follows the button in `base.html` is not included when the codename hint is displayed in `/lookup`. The changes in https://github.com/freedomofpress/securedrop/pull/5096/commits/08edfa2c198ecbdf3e14895f4aa05d56069c861c restores the button while keeping the `/lookup` layout as desired.